frontend/src/app/modules/hal/resources/work-package-resource.ts

```diff
@@ -182,6 +183,10 @@ export class WorkPackageBaseResource extends HalResource {
     return otherWorkPackage.parent?.$links.self.$link.href === this.$links.self.$link.href;
   }
 
+  public tabs(registeredTabs: Tab[]):Tab[] {
+    return _.filter(registeredTabs, (tab) => tab.displayable(this));
```

Comment:
I'm not too happy with the `tabs` being added to the resource itself. It doesn't really have anything to do with the work package resource itself. It could just be a private member of the `TabWrapperComponent` ?

Reply:
yep, it's using the `workPackage` resource as kind of a helper module. I think, since the implementation became quite simple with the recent refactorings, that we can just inline the implementation.